### PR TITLE
Add depth anything and object detection sensory agent.

### PR DESCRIPTION
**Pull request by nqyy**
Two more agents are added:
- Depth estimation
- object detection

### CODE REVIEW

File: mbodied/agents/sense/depth_estimation_agent.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        model_kwargs=None,
+        **kwargs,
+    ):
+        super().__init__(
+            model_src=model_src,
+            model_kwargs=model_kwargs,
+            **kwargs,
+        )
+
+    def act(self, image: Image, *args, api_name: str = "/depth", **kargs) -> Image:
```
Comment (sebbyjp) [inline mbodied/agents/sense/depth_estimation_agent.py:39]: fix kargs type please for here and below

File: mbodied/types/sense/scene.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """Model for Scene Object Poses.
+
+    Attributes:
+        name (str): The name of the object.
+        pose (Pose): The pose of the object.
+        bbox_3d (NumpyArray | None): The 3D bounding box of the object.
+        pixel_coords (NumpyArray | None): The pixel coordinates of the object.
+        bbox_pixels (NumpyArray | None): The pixel coordinates of the bounding box.
+    """
+
```
Comment (sebbyjp) [inline mbodied/types/sense/scene.py:21]:
```suggestion
class BBox2D(NamedTuple):
  x1: float
  y1: float
  x2: float
  y2: float
```
PixelCoords(NamedTuple):
  u: int
  v: int

BBox3D... etc

**Review by sebbyjp — COMMENTED** [review]
add to pyproject.toml please
[tool.pytest.ini_options]
addopts = -m "not slow" -m "not network"
markers = [
    "slow: marks tests as slow (deselect with '\"-m "not slow\"')",
    "network": network calls (deselect with '\"-m "not network\"')",
]

File: mbodied/agents/sense/object_detection_agent.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        model_kwargs=None,
+        **kwargs,
+    ):
+        super().__init__(
+            model_src=model_src,
+            model_kwargs=model_kwargs,
+            **kwargs,
+        )
+
+    def act(self, image: Image, objects: list[str] | str, *args, api_name: str = "/detect", **kwargs) -> Scene:
```
Comment (sebbyjp) [inline mbodied/agents/sense/object_detection_agent.py:35]: I thought we are calling this endpoint?

**Review by sebbyjp — CHANGES_REQUESTED** [review]
Waiting for the pytest.ini

**Comment by nqyy** [comment]
All resolved.

**Review by sebbyjp — APPROVED** [review]

**Pull request merged**